hack/lib/swagger.sh

```diff
@@ -59,5 +59,5 @@ EOF
   echo "// AUTO-GENERATED FUNCTIONS END HERE" >> "${TMPFILE}"
 
   gofmt -w -s "${TMPFILE}"
-  mv "${TMPFILE}" ""${gv_dir}"/types_swagger_doc_generated.go"
```

Comment:
this quoting was totally wrong but if your path didn't have spaces it would work out